### DISCUSSION TITLE
Gradually depreciating URLs

### DISCUSSION THREAD

**Discussion by wes-otf** [OpenTechFund/opentech.fund#4171] [General]
Some feedback I recently received from staff was that they had bookmarked URLs with specific filters from the beta view, then once that was changed from beta -> list view, they needed to manually fix all their bookmarks. There are also links in internal docs that have a potential to break as new features are iterated on & things are renamed.

As Hypha's user base grows & different users save/share URLs for different reasons, it may be worthwhile to support redirects from old URLs to new a period of time (giving users a toast notification that the URL would be removed soon?)

This wouldn't be worth it for most URL spaces that users wouldn't typically interact with, but something to consider moving forward.

**Comment by frjo** [comment]
Good point!
